test/src/helpers/school-set-up-utils.js

```diff
@@ -0,0 +1,37 @@
+const {getToken} = require('./oauth-utils');
+const {getSchoolIDBySchoolCode} = require('../services/institute-api-service');
+
+const constants = require('../config/constants');
+const restUtils = require('./rest-utils');
+import {DateTimeFormatter, LocalDate} from '@js-joda/core';
+
+const schoolSetUpUtils = {
+
+    async getSchoolDetails(schoolNumber){
+
+        const data = await getToken();
+        const token = data.access_token;
+        let schoolID = await getSchoolIDBySchoolCode(schoolNumber);
+        const url = `${constants.institute_base_url}/api/v1/institute/school/${schoolID}`;
+        return await restUtils.getData(token, url);
+    },
+    async getSchoolPrincipalDetails(schoolNumber){
+        let schoolPrincipal = '';
+        let schoolDetails = await schoolSetUpUtils.getSchoolDetails(schoolNumber);
+        for (const schoolContact of schoolDetails.contacts){
+            if(schoolContact.schoolContactTypeCode === 'PRINCIPAL'){
+                let parsedExpiryDate = null;
+                if (schoolContact.expiryDate) {
+                    parsedExpiryDate = new LocalDate.parse(schoolContact.expiryDate, DateTimeFormatter.ofPattern('uuuu-MM-dd\'T\'HH:mm:ss'));
```

Comment:
after yesterday's date fiasco, we probably want to use LocalDateTime. LocalDate has funny behaviors with timestamps.  HH:mm:ss